virtcontainers_test.go

```diff
@@ -102,6 +103,7 @@ func TestMain(m *testing.M) {
 	podDirLock = filepath.Join(runStoragePath, testPodID)
 	podFileState = filepath.Join(runStoragePath, testPodID, stateFile)
 	podFileLock = filepath.Join(runStoragePath, testPodID, lockFileName)
+	testBinHookPath = filepath.Join(testDir, "bin", "hook")
```

Comment:
Unfortunately, this won't work. Look at travis.yml, we are building and copying the hook binary at that moment, meaning that /tmp/bin/hook is a hardcoded value. The dynamic directory created by ioutil.TempDir() won't fit.